server/controllers/calls.js

```diff
@@ -238,3 +264,22 @@ export function hangUpCall(req, res) {
       res.status(500).json({ message: 'could not get user\'s by ID ' });
     });
 }
+
+/*
```

Comment:
Do we need to keep this commented code around?

Reply:
oops, nope, I removed it.